lib/hexpm_web/views/email_view.ex

```diff
@@ -1,3 +1,57 @@
 defmodule HexpmWeb.EmailView do
   use HexpmWeb, :view
+
+  defmodule OwnerAdd do
+    def message(package, version) do
+      "#{username} has been added as an owner to package #{package}."
+    end
+  end
+
+  defmodule OwnerRemove do
+    def message(package, version) do
+      "#{@username} has been removed from owners of package #{@package}."
+    end
+  end
+
+  defmodule Verification do
+    def intro() do
+      "To begin using your email, we require you to verify your email address."
+    end
+
+    def verify_links(verify_url, :html) do
+      "You can do so by following <a href=\"#{raw verify_url}\">this link</a> or by pasting this link in your web browser: #{raw verify_url}"
+    end
+    def verify_links(verify_url, :text) do
```

Comment:
I am not overly happy with this portion, having to specify multiple functions for the same thing. If anyone has a suggestion for something like this, it would be appreciated.